docs/ARRIBA_2.2.1.md

```diff
@@ -0,0 +1,47 @@
+# Arriba v2.2.1
+"[Arriba](https://arriba.readthedocs.io/en/v2.2.1/) is a command-line tool for the detection of gene fusions from RNA-Seq data.
+It was developed for the use in a clinical research setting.
+Therefore, short runtimes and high sensitivity were important design criteria"
+It runs in about 30 minutes - 2 hours costing about $0.17 - $0.86 on a `r5.2xlarge` spot instance.
+
+## Fusion detection
+`tools/arriba_fusion_2.2.1.cwl`
+### Required references:
+ - `reference_fasta`: Fasta reference file used for alignment
+ - `gtf_anno`: GTF file used for alignment indexing
+### Required inputs:
+ - `genome_aligned_bam`: STAR-aligned, **coordinate sorted** bam file
+ - `outFileNamePrefix`: String to prepend output file names with
+## Optional inputs:
```

Comment:
Should Required and Optional inputs be on the same level?

Reply:
Good call, should be